src/furtherWork/gameProperties.js

```diff
@@ -0,0 +1,64 @@
+//generate random species for the groot
+
+const grootTypes = [
+   'adder',
+   'agama',
+   'blind snake',
+   'blue racer',
+   'boa constrictor',
+   'bushmaster',
+   'diamond python',
+   'habu',
+   'whip snake',
+   'viper',
+   'sidewinder',
+   'ringhals',
+   'python',
+   'pit viper',
+   'massasauga'
+   ];
+
+// generate a battle cry for the groot - to be used when new pet is created and when an attack is made
```

Comment:
Should be removed. This is the type of info that would be great to put in the readme, showcase what youve made and show it off as soon as someone lands in your repo!